pom.xml

```diff
@@ -190,6 +191,24 @@
                 <artifactId>junit</artifactId>
                 <version>4.12</version>
             </dependency>
+
+            <dependency>
+                <groupId>io.netty</groupId>
+                <artifactId>netty-handler</artifactId>
+                <version>${dep.netty.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>io.netty</groupId>
+                <artifactId>netty-buffer</artifactId>
+                <version>${dep.netty.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>io.netty</groupId>
+                <artifactId>netty-transport</artifactId>
+                <version>${dep.netty.version}</version>
```

Comment:
Why do you need netty? I don't see it used anywhere. No other pom.xml (modules) is using that.

Reply:
Now that kafka_2_12 is removed it's not needed, thanks!